### ISSUE TITLE
Add support for 3rd party captcha services

### ISSUE THREAD

**Issue by wertyy102** [pterodactyl/panel#1895]
**Is your feature request related to a problem? Please describe.**
Google's reCAPTCHA is good and all, but Cloudflare made the decision to switch, and I feel that we should also be able to use 3rd party Captcha services with pterodactyl.

https://blog.cloudflare.com/moving-from-recaptcha-to-hcaptcha/

**Describe the solution you'd like**
I would like support for hCaptcha on pterodactyl
https://www.hcaptcha.com/

**Describe alternatives you've considered**
Any alternative Captcha setup, or a form where we can fill in the various links and related parts to set up a captcha service of our choosing.

**Comment by matthewpi** [comment]
While I agree more options is better, there are a few problems I see with this.

Adding more captcha options will most likely be a pain in 0.8 because of the front-end being rendered client-side compared to server-side in 0.7 (current stable release of the panel), meaning we would have to figure out what captcha script to load on the client without the server being able to just change a script tag.

It also probably isn't a good idea to pre-load more than one captcha library or have the panel return a url to a script that should be loaded (for obvious CSRF reasons and wasting bandwidth by loading more scripts than the client needs)

Being able to change captcha services would probably be a very low use feature, many people don't even setup custom reCaptcha keys on the panel and there are still a lot of people who would still use reCaptcha over other services.

TL;DR: It may happen but don't place your bets on it, if it is to happen it would be low on the list because there are other features that have been highly requested ages ago and it would be nice to get those implemented for the release of 0.8.

**Comment by DaneEveritt** [comment]
I won't be implementing logic to handle custom captchas at this time. I am open to re-evaluating this decision down the road if necessary.
